com.vrcfury.vrcfury/Editor/VF/Inspector/VRCFuryActionEditor.cs

```diff
@@ -108,6 +108,60 @@ public class VRCFuryActionDrawer : PropertyDrawer {
 
                 return row;
             }
+            case nameof(PoiyomiUVTileAction): {
+                var content = new VisualElement();
+                var row = new VisualElement {
+                    style = {
+                        alignItems = Align.FlexStart,
+                        flexDirection = FlexDirection.Row,
+                    }
+                };
+
+                var label = new Label("Poiyomi UV Tile Discard") {
+                    style = {
+                        flexGrow = 0,
+                        flexBasis = 140,
+                    }
+                };
+                row.Add(label);
+
+                var propField = VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("renderer"));
+                propField.style.flexGrow = 1;
+                row.Add(propField);
+            
+                var propField2 = VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("slot1"), tooltip: "UV X Axis, Val 0-3");
+                propField2.style.flexGrow = 0;
+                propField2.style.flexBasis = 30;
+                row.Add(propField2);
+
+                var propField3 = VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("slot2"),tooltip: "UV Y Axis, Val 0-3");
+                propField3.style.flexGrow = 0;
+                propField3.style.flexBasis = 30;
+                row.Add(propField3);
+                content.Add(row);
+
+                var row2 = new VisualElement {
+                    style = {
+                        alignItems = Align.FlexStart,
+                        flexDirection = FlexDirection.Row,
+                    }
+                };
+
+                var adv = new Foldout {
+                    text = "Advance Options",
+                    style = {
+                        flexDirection = FlexDirection.Column,
+                    },
+                    value = false
+                };
+                adv.Add(VRCFuryEditorUtils.Prop(prop.FindPropertyRelative("invert"), "Invert the UV Tile Discarding."));

```

Comment:
The description of this should probably be something like "Invert (Discard the UVs when active)"